02-express-tutorial/app.js

```diff
@@ -1 +1,42 @@
 console.log('Express Tutorial')
+const express = require('express')
+const { products } = require('./data.js')
+const app = express()
+app.use(express.static("./public"))
+const port = 4000
+
+
+app.get("/api/v1/test", (req, res) => {
+    res.json({ message: "It worked!" });
+})
+app.get("/api/v1/products", (req, res) => {
+    res.json(products);
+})
+
+app.get("/api/v1/products/:productID", (req, res) => {
+    const id = req.params.productID;
+    if(!isNaN(id)) return res.json({ message: "Invalid product ID" });
```

Comment:
isNaN returns true if the value is Not a Number so it should not be negated
 if(isNaN(id)) return res.json({ message: "Invalid product ID" });